BillManagementSystem/accounting/schema/schema.py

```diff
@@ -0,0 +1,69 @@
+import strawberry
+from typing import List
+from django.contrib.auth.models import User as AdminUser
+from .inputs import CategoryInput, CategoryPartialInput, RegisterInput, BillInput, BillPartialInput, IndividualSpendingInput, IndividualSpendingPartialInput
+from strawberry_django import mutations
+from .. import models
+from django.contrib.auth.tokens import default_token_generator
+
+from .types import IndividualSpendingScalar, BillScalar, CategoryScalar, UserScalar, AuthResponse
+from .filters import UserFilter, BillFilter, IndividualSpendingFilter, CategoryFilter
+
+from gqlauth.user import arg_mutations as mutationsAuth
+
+@strawberry.type
+class Query:
+    categories: List[CategoryScalar] = strawberry.django.field()
+    category: List[CategoryScalar] = strawberry.django.field(filters=CategoryFilter)
+    
+    users: List[UserScalar] = strawberry.django.field()
+    user: List[UserScalar] = strawberry.django.field(filters=UserFilter)
+
+    bills: List[BillScalar] = strawberry.django.field()
+    bill: List[BillScalar] = strawberry.django.field(filters=BillFilter)
+    
+    individualSpendings: List[IndividualSpendingScalar] = strawberry.django.field()
+    individualSpending: List[IndividualSpendingScalar] = strawberry.django.field(filters=IndividualSpendingFilter)
+    
```

Comment:
Is there any difference between the plural and singular forms of these?
If the plural one is to list all, it is not necessary, as even if you declare filters, it will not be activated unless you pass a value.